docs/plugins/modules/netbox_device_interface/netbox.netbox.netbox_device_interface_module.rst

```diff
@@ -81,7 +81,7 @@ Parameters
                                 <td>
                                                                                                                                                             </td>
                                                                 <td>
-                                            <div>Defines the prefix configuration</div>
+                                            <div>Defines the interface configuration</div>
```

Comment:
These are actually generated from the modules themself.

Revert these changes and update within the module itself and then I typically generate new docs on each release and these changes will be made in these `.rst` files.

Reply:
I updated the changes in the docstring in the module and reverted the changes to the docs file.